extensions/stackdriver/log/logger.cc

```diff
@@ -72,6 +71,16 @@ Logger::Logger(const ::wasm::common::NodeInfo& local_node_info,
   (*label_map)["destination_workload"] = local_node_info.workload_name();
   (*label_map)["destination_namespace"] = local_node_info.namespace_();
   (*label_map)["mesh_uid"] = local_node_info.mesh_id();
+  // Add destination app and version label if exist.
+  const auto& local_labels = local_node_info.labels();
+  auto version_iter = local_labels.find("version");
+  if (version_iter != local_labels.end()) {
+    (*label_map)["destination_version"] = version_iter->second;
+  }
+  auto app_iter = local_labels.find("app");
+  if (app_iter != local_labels.end()) {
+    (*label_map)["destination_app"] = app_iter->second;
```

Comment:
Note somewhere that this is needed by stack driver to correlate with platform logs.

Reply:
Done. add a log about it.